src/components/header/Header.tsx

```diff
@@ -1,2 +1,57 @@
+import cartIcon from '../../assets/cart.png';
+import dividerIcon from '../../assets/divider.png';
+import logoMA from '../../assets/logoMA.png';
+import logoutIcon from '../../assets/logout.png';
+import moonIcon from '../../assets/moon.png';
+import sunIcon from '../../assets/sun.png';
+import useraddIcon from '../../assets/useradd.png';
+import Button from '../Button/Button.tsx';
+import Image from '../Image/Image.tsx';
+import Link from '../Link/Link.tsx';
+
 import styles from './header.module.css';
-export const Header = () => <header className={styles.header}>Header</header>;
+export const Header = () => (
+    <header className={styles.header}>
+        <div className={styles.headerBox}>
+            <div className={styles.headSection}>
+                <Image src={logoMA} alt="logo" className="headerLogo"></Image>
+                <div className={styles.colorSwitch}>
+                    <Image src={sunIcon} alt="sunPic" className={styles.headerSun}></Image>
+                    <Image src={dividerIcon} alt="dividerPic" className={styles.headerDivider}></Image>
+                    <Image src={moonIcon} alt="moonPic" className={styles.headerMoon}></Image>
```

Comment:
if u will replasce your png icons into svg this will be more readable because y will have
```suggestion
                    <SunIcon  className={styles.headerMoon} />
                    <DividerIcon  className={styles. headerDivider} />
                    <MoonIcon  className={styles. headerMoon} />
```

Reply:
here too